code/game/machinery/dna_infuser/dna_infuser.dm

```diff
@@ -175,6 +208,9 @@
 		return FALSE
 	if(isliving(target))
 		var/mob/living/living_target = target
+		if(iscarbon(target))
+			balloon_alert(user, "no higher-level creatures!")
```

Comment:
i dont think this message will ever happen because you early return on iscarbon(target) here https://github.com/tgstation/tgstation/pull/71605/files#diff-1c4d82214ef35225bec0d8e70fde3e713c1a120d13c3bac7c348024c416367d6R195 before calling this proc. and the only other call of this proc is in attackby()

Reply:
nice catch